### PR TITLE
Typographical changes only.

### PR DESCRIPTION
**Pull request by sashamitelman**
@mfine Pushing my initial fixes on typos in SBP documentation.

### CODE REVIEW

File: spec/yaml/swiftnav/sbp/tracking.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ definitions:
             type: u32
             desc: |
               Signal identifier being tracked - values 0x00 through 0x1F
-              represent GPS PRNs 1 through 32 respectively (PRN-1 notation);
+              represent GPS PRNs 1 through 32 respectively (PRN-minus-1 notation);
```
Comment (mfine) [inline spec/yaml/swiftnav/sbp/tracking.yaml]: `PRN-1` is being used in other places as well:

```shell
yaml/swiftnav/sbp/acquisition.yaml
47:              through 32 respectively (PRN-1 notation); other values reserved
74:              PRN-1 identifier of the satellite signal for which

yaml/swiftnav/sbp/observation.yaml
93:              (PRN-1 notation); other values reserved for future use.
250:            GPS PRNs 1 through 32 respectively (PRN-1 notation); other values
507:            desc: PRN-1 identifier of the satellite signal

yaml/swiftnav/sbp/tracking.yaml
41:              represent GPS PRNs 1 through 32 respectively (PRN-1 notation);
87:            represent GPS PRNs 1 through 32 respectively (PRN-1 notation);
114:            desc: PRN-1 being tracked
```

Should we grab those too?

File: spec/yaml/swiftnav/sbp/tracking.yaml [hunk 2]
```diff
@@ -50,7 +50,7 @@ definitions:
     short_desc: Satellite tracking channel states
     desc: |
       The tracking message returns a variable-length array of tracking
-      channel states. It reports status and snr power measurements for
+      channel states. It reports status and C/N0 measurements for
```
Comment (mfine) [inline spec/yaml/swiftnav/sbp/tracking.yaml]: We haven't use `C/N0` in the documentation - have used instead `Carrier-to-Noise density`.

File: spec/yaml/swiftnav/sbp/tracking.yaml (hunk 1 above)
Comment (sashamitelman) [inline spec/yaml/swiftnav/sbp/tracking.yaml]: Absolutely. Good catch. I'd flagged those previously but just hadn't made the changes in my local repo.

I'll do that now and the changes to the existing PR.

**Comment by mfine** [comment]
Couple of nits, looks good otherwise. Also feel free to generated a new PDF.

**Comment by sashamitelman** [comment]
@mark Made the last of the typo changes, and built PDF. Can you verify, then I'll merge this PR.

**Comment by mfine** [comment]
Looks good - feel free to squash down the commits and then merge.

**Pull request merged**